.cspell.json

```diff
@@ -0,0 +1,61 @@
+{
+  "version": "0.2",
+  "words": [
+    "AAAAAEAAQAAACI",
```

Comment:
I think you sometimes get these with unignored image files

Reply:
yep, do you know of a good regex we can use so we don't need to include it here?

Reply:
If you can figure out what file it's coming from, you could add the file type to the `ignorePaths` below like `"**/*.png"`

Reply:
it's this one:
![image](https://user-images.githubusercontent.com/1212885/189810835-1a07ed48-7f6e-4d45-ba38-54e80b8b1521.png)

Reply:
Ah, there are some built in `ignoreRegExpList` like `"Urls"` that may be able to disable those ones. Not sure if it would trip on the internal string or not